packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -189,6 +167,28 @@ class AndroidDevice extends Device {
     return <String>[getAdbPath(androidSdk), '-s', id]..addAll(args);
   }
 
+  String runAdbMostlyCheckedSync(
```

Comment:
runAdbMostlyChecked -> runAdbChecked to avoid a rename when the whitelist is no longer needed.